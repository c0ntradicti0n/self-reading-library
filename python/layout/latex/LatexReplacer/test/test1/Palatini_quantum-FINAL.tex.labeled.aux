\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\citation{1207.7214,1207.7235}
\citation{Englert:1964et,Higgs:1964ia,Higgs:1964pj}
\citation{1001.4538,1807.06211}
\citation{Starobinsky:1980te,Guth:1980zm,Linde:1981mu,Mukhanov:1981xt}
\citation{0710.3755}
\citation{0710.3755}
\citation{0803.2664}
\citation{1807.02376}
\citation{2001.10135}
\citation{Callan:1970ze}
\citation{0710.3755}
\citation{Palatini19,Einstein25}
\citation{0803.2664}
\citation{0803.2664}
\citation{1807.06211}
\citation{1303.3787}
\citation{1008.5157}
\citation{0902.4465,0903.0355}
\citation{1012.2900}
\citation{1003.2635,1005.2978}
\citation{1003.2635}
\citation{1612.06253,1711.08761}
\citation{1008.5157}
\citation{1609.05209,1610.08916}
\citation{1008.5157,1412.3811}
\citation{1609.05209,1610.08916}
\citation{0812.3622,0812.4624}
\citation{1902.10148}
\citation{1010.1417,1307.5298,1501.02231,1701.07665}
\citation{1008.5157,Aydemir:2012nz}
\citation{1008.5157}
\citation{1412.3811}
\citation{1008.5157,1403.6078,1412.3811,Enckell:2016xse,1706.05007}
\citation{0904.1537}
\citation{0812.4950,0904.1537,1411.1923}
\citation{0809.2104,0812.4946,0904.1698,0910.1041}
\citation{1709.07853,1712.04874,1802.09299}
\citation{1012.2900}
\@writefile{toc}{\contentsline {section}{\numberline {1} section section }{1}{section.1}}
\newlabel{sec:introduction}{{1}{1}{section section}{section.1}{}}
\citation{0710.3755}
\citation{0803.2664}
\citation{0812.4624}
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces  caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption \relax }}{4}{table.caption.2}}
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{table}{{1}{4}{caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption \relax }{table.caption.2}{}}
\@writefile{toc}{\contentsline {section}{\numberline {2} section section section section section }{4}{section.2}}
\newlabel{sec:review}{{2}{4}{section section section section section}{section.2}{}}
\citation{1807.06211}
\citation{1609.05209,1610.08916,1902.10148}
\citation{1906.11837}
\citation{1902.10148}
\citation{1807.06211}
\citation{1008.5157,1412.3811}
\citation{1008.5157,1412.3811}
\citation{tHooft:1972tcz}
\citation{1412.3811}
\newlabel{slowRollParameters}{{2.1}{5}{section section section section section}{equation.2.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {3} section section section section section }{5}{section.3}}
\newlabel{sec:newPhysics}{{3}{5}{section section section section section}{section.3}{}}
\citation{1412.3811}
\citation{Bardeen:1995kv,0708.3550,1303.7244}
\citation{1308.2627,1402.1476}
\citation{1712.08471,1904.05699}
\citation{1712.04874}
\@writefile{lot}{\contentsline {table}{\numberline {2}{\ignorespaces  caption caption caption caption caption caption caption caption caption caption \relax }}{7}{table.caption.3}}
\newlabel{table2}{{2}{7}{caption caption caption caption caption caption caption caption caption caption \relax }{table.caption.3}{}}
\citation{1507.08833}
\citation{1205.2892}
\citation{1205.2893,1205.6497,1307.3536}
\citation{0904.1537}
\citation{Tanabashi:2018oca}
\citation{1709.07853,1802.09299}
\@writefile{toc}{\contentsline {section}{\numberline {4} section section section section section section }{8}{section.4}}
\newlabel{sec:connection}{{4}{8}{section section section section section section}{section.4}{}}
\citation{1709.07853}
\citation{1709.07853}
\citation{1509.04044}
\citation{1810.01772}
\citation{Tanabashi:2018oca}
\citation{1411.1923}
\citation{1412.3649,1801.03944}
\citation{0812.4950,0904.1537}
\citation{1008.5157,1403.6078,1412.3811}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces  caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption \relax }}{10}{figure.caption.4}}
\newlabel{sfig:xiOfLambda0}{{1}{10}{caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption \relax }{figure.caption.4}{}}
\citation{0710.3755}
\citation{0803.2664}
\citation{1012.2900}
\citation{1008.5157}
\citation{1904.05237,1905.02302}
\citation{1904.05237}
\citation{1905.02302}
\citation{Froggatt:1995rt}
\citation{0912.0208}
\citation{2001.09088}
\citation{2001.09088}
\citation{Asaka:2005an,Asaka:2005pn}
\citation{1005.3497,1010.1415,1103.5963}
\@writefile{toc}{\contentsline {section}{\numberline {5} section section }{11}{section.5}}
\newlabel{sec:conclusion}{{5}{11}{section section}{section.5}{}}
\citation{mathematica}
\@writefile{toc}{\contentsline {section}{\numberline {A} section section section section }{12}{appendix.A}}
\newlabel{app}{{A}{12}{section section section section}{appendix.A}{}}
\bibstyle{JHEP}
\bibdata{inflationBib}
\@writefile{lof}{\contentsline {figure}{\numberline {2}{\ignorespaces  caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption Fig.\spacefactor \@m {}\xspace  \ref  {sfig:xiOfLambda0} caption \relax }}{13}{figure.caption.6}}
\newlabel{sfig:epsilonOfLambda0}{{2}{13}{caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption caption \fig \ref {sfig:xiOfLambda0} caption \relax }{figure.caption.6}{}}
