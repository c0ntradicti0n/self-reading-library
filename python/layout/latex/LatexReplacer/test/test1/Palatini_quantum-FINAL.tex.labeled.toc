\contentsline {section}{\numberline {1} section section }{1}{section.1}
\contentsline {section}{\numberline {2} section section section section section }{4}{section.2}
\contentsline {section}{\numberline {3} section section section section section }{5}{section.3}
\contentsline {section}{\numberline {4} section section section section section section }{8}{section.4}
\contentsline {section}{\numberline {5} section section }{11}{section.5}
\contentsline {section}{\numberline {A} section section section section }{12}{appendix.A}
