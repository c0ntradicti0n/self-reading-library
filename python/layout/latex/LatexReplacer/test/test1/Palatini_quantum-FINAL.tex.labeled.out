\BOOKMARK [1][-]{section.1}{ section section }{}% 1
\BOOKMARK [1][-]{section.2}{ section section section section section }{}% 2
\BOOKMARK [1][-]{section.3}{ section section section section section }{}% 3
\BOOKMARK [1][-]{section.4}{ section section section section section section }{}% 4
\BOOKMARK [1][-]{section.5}{ section section }{}% 5
\BOOKMARK [1][-]{appendix.A}{ section section section section }{}% 6
